docs/src/jsx/utils/CodeSandboxLink.js

```diff
@@ -0,0 +1,70 @@
+//  Copyright (c) 2018-present, GM Cruise LLC
+//
+//  This source code is licensed under the Apache License, Version 2.0,
+//  found in the LICENSE file in the root directory of this source tree.
+//  You may not use this file except in compliance with the License.
+
+import LZString from "lz-string";
+import React from "react";
+import styled from "styled-components";
+
+import CodeSandboxIcon from "./icons/CodeSandbox";
+import { palette } from "./theme";
+
+function compress(string) {
+  return LZString.compressToBase64(string)
+    .replace(/\+/g, `-`) // Convert '+' to '-'
+    .replace(/\//g, `_`) // Convert '/' to '_'
+    .replace(/=+$/, ``); // Remove ending '='
```

Comment:
Why are these replaces here? A comment would be helpful to explain the reasoning.

Reply:
Good catch. it's not really necessary there. I was using the GET request in the beginning which has some restrictions on how urls are formatted. Later when I switched to the POST request, I forgot to remove these string manipulations. I'l just remove to avoid confusion.